hs_access_control/models.py

```diff
@@ -1297,7 +1298,7 @@ def can_view_resource(self, this_resource):
 
         access_resource = this_resource.raccess
 
-        if access_resource.public:
+        if access_resource.public or access_resource.published:
```

Comment:
If a resource has been published, it must be public. This change to check for 'published' is not needed.